enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -1012,8 +1012,14 @@ class Meta:
     def save(self, *args, **kwargs):
         """
         This type of policy must always have an access_method of "assigned".
+        Additionally, if this record is being created, and no ``assignment_configuration``
+        is present, create one and associate it with this record.
         """
         self.access_method = AccessMethods.ASSIGNED
+        if self._state.adding and not self.assignment_configuration:
```

Comment:
Nit: if self._state.adding is ORM magic, please add a comment explaining what it does.